src/UglyToad.PdfPig/Filters/CcittFaxDecodeFilter.cs

```diff
@@ -2,18 +2,26 @@
 {
     using System;
     using System.Collections.Generic;
+    using System.Linq;
     using Tokens;
 
     internal class CcittFaxDecodeFilter : IFilter
     {
         /// <inheritdoc />
-        public bool IsSupported { get; } = false;
-
+        public bool IsSupported { get; } = true;
+        
         /// <inheritdoc />
         public byte[] Decode(IReadOnlyList<byte> input, DictionaryToken streamDictionary, int filterIndex)
         {
-            throw new NotSupportedException("The CCITT Fax Filter for image data is not currently supported. " +
-                                            "Try accessing the raw compressed data directly.");
+            if (input == null)
```

Comment:
I think the expected output here would be to convert the image data to a bitmap similar to https://github.com/apache/pdfbox/blob/trunk/pdfbox/src/main/java/org/apache/pdfbox/filter/CCITTFaxFilter.java

I'll merge this change but I think I'll remove this filter implementation and update the PdfImageHelper implementation to be (see next comment)